quicksort2/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"fmt"
+)
+
+func qsort(a []int) []int {
+	if len(a) < 2 { return a }
+
+
+	biggestnum := a[len(a)/2]
+
+
+
+
+	var rightslice []int
+	for i := range a {
+		if a[i] > biggestnum {
+			biggestnum = a[i]
+		}
+	}
```

Comment:
we can little change the code:

```go
for _, v := range a {
		if v > biggestNum {
			biggestNum = v
		}
	}
```

In `for` loop we use underscore to skip index of item (because it is no more used). And `v` is actual the value of array.